### ISSUE TITLE
Cluster Guide Doc Basic Example Discrepency

### ISSUE THREAD

**Issue by kaustav202** [mingrammer/diagrams#753]
The output image of basic cluster example [ https://diagrams.mingrammer.com/docs/guides/cluster#basic ] does not match with resource names in source code 

RDS1 : shown -> primary , actual -> master
RDS2 : shown -> replica1 , actual -> slave1
RDS3 : shown -> replica2 , actual -> slave2

**Comment by gabriel-tessier** [comment]
Good catch @kaustav202 , my bad I forget to update the png file in the doc when I made the changed.
I pushed a PR https://github.com/mingrammer/diagrams/pull/790.
